### ISSUE TITLE
SAMOVAR Stand-Off OECM Pod

### ISSUE THREAD

**Issue by FrangibleCover** [PygmalionOfCyprus/cmo-db-requests#2474]
### DB Selector

DB3K

### Baseline

Weapon 1860, 150kg

### Hypothetical

No

### Name

SAMOVAR

### Sensor Generation

Early 1990s

### Range (nm)

-

### Sensor Capabilities

Torpedo Warning

### Search/Track Frequencies

I Band (8-10 GHz) [Old X Band] 3-3.75 cm, J Band (10-20 GHz) [Old X Band] 1.5-3 cm

### Lower Frequency (Hz)

8 GHz

### Upper Frequency (Hz)

16 GHz

### Comments

An unusual Norwegian jammer, deployed on the F-5 fleet as dedicated Stand-Off/Stand-In OECM aircraft and also on the F-16 fleet, possibly in the same role or possibly alongside Penguins. SAMOVAR (SAM Obstruction in Velocity, Angle and Range) was designed to support anti-ship strikes and has a very narrow band for an OECM system, aimed squarely at disrupting fire control radars.

The jammer operates in CW, Interrupted CW and Pulse modes and entered service in 1994, at the same time as the first F-5 PAWS upgrades. Some sources call it an ALQ-300, but since it's an indigenous Norwegian system I don't think it'd have an ALQ designation.

### Sources

https://no.m.wikipedia.org/wiki/Samovar_(elektronisk_krigf%C3%B8ring)
https://www.thefreelibrary.com/Farnborough+highlights+international+electronic+warfare-a016340442
https://www.the-northrop-f-5-enthusiast-page.info/AirForces/NorwayUnits.html
Various Janes articles

**Comment by nuk92** [comment]
I think you got the upper frequency wrong, I found 16ghz in your sources.

**Comment by FrangibleCover** [comment]
Good catch, thanks. Can't wait for this one to be honest, I think it'll significantly change the complexion of the early Northern Fury scenarios and who doesn't love Northern Fury?

**Comment by nuk92** [comment]
To be honest I didn't have much time to play most scenarios but still can't wait for these requests to be implemented :smiley:
